reframe/core/launchers/mpi.py

```diff
@@ -124,3 +124,26 @@ def command(self, job):
             ret.append(opt)
 
         return ret
+
+
+@register_launcher('lrun')
+class LrunLauncher(JobLauncher):
+    '''LLNL's custom parallel job launcher.'''
+    def command(self, job):
+        num_tasks = job.num_tasks or 1
+        num_tasks_per_node = job.num_tasks_per_node or 1
+        num_nodes = int(num_tasks // num_tasks_per_node)
+        return ['lrun', '-N', str(num_nodes),
+                '-T', str(num_tasks_per_node)]
+
+
+@register_launcher('lrun-gpu')
+class LrungpuLauncher(JobLauncher):
+    '''LLNL's custom parallel job launcher w/ CUDA aware Spectum MPI.'''
+    def command(self, job):
+        num_tasks = job.num_tasks or 1
+        num_tasks_per_node = job.num_tasks_per_node or 1
+        num_nodes = int(num_tasks // num_tasks_per_node)
+        return ['lrun', '-M "-gpu"',
+                '-N', str(num_nodes),
+                '-T', str(num_tasks_per_node)]
```

Comment:
You could possibly inherit from `LrunLauncher()` and implement `command()` as follows:

```suggestion
    def command(self, job):
        return super().command(job) + ['-M "-gpu"']
```

Reply:
done